agent/util-scripts/pbench-tool-meister-start

```diff
@@ -340,6 +342,18 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    try:
+        tm_start_path = Path(
+            Path(__file__).resolve(), "..", ".."
```

Comment:
We need to consider pulling the installation directory path from a `PbenchAgentConfiguration` object instead.

Reply:
Is this an existing class or something else to be implemented?

Reply:
Existing, see https://github.com/distributed-system-analysis/pbench/blob/tool-meister/lib/pbench/agent/__init__.py#L14.